### ISSUE TITLE
Move Technical Charter to this repo

### ISSUE THREAD

**Issue by vmbrasseur** [tungstenfabric/docs#9]
Currently the technical charter is [in a Google Doc](https://docs.google.com/document/d/1dszbUJwwQoEffY1ekn1yMpBiDNOe6hO4Nshwp8qYSLE/edit).

Convert it to Markdown and move it to this repo.

**Comment by vmbrasseur** [comment]
Plan of attack:

* Download in docx format
* Run pandoc on it, docx -> gfm (GitHub Flavoured Markdown)
* Verify it still looks OK & iterate if necessary
* Send PR

**Comment by vmbrasseur** [comment]
Update: Will convert to rst instead, since it's better able to handle the type of ordered list numbering that we'll need if we want to maintain the numbering that's in the current doc.

See [Richard's rst primer](http://docutils.sourceforge.net/docs/user/rst/quickstart.html#lists) for more information.

**Comment by vmbrasseur** [comment]
The docx file needed a bunch of cleanup to make sure that the ordered lists were correct.

Once that was complete and confirmed against the Google Doc, I ran the docx through pandoc with this command:

```
pandoc -f docx TungstenFabricProjectTechnicalCharter.docx -t rst -o TungstenFabricProjectTechnicalCharter.rst
```

I previewed the rst using [this online tool](http://rst.ninjs.org) and discovered that the ordered lists were no longer correct (the lowercase alphabet items didn't restart at 'a' each time, they just kept going through the alphabet). I fixed those manually in vim.

I think this is now ready for approval and merging into the docs directory, or will be once I send a PR.
